paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/FormFragmentArguments.kt

```diff
@@ -16,6 +16,7 @@ internal data class FormFragmentArguments(
     val merchantName: String,
     val amount: Amount? = null,
     val billingDetails: PaymentSheet.BillingDetails? = null,
+    val requiresUserOptInToSavePaymentMethod: Boolean = false,
```

Comment:
Is this variable used somewhere?

Reply:
Good point. I failed to notice David added it to the iOS playground. I just pushed a commit to add it to ours.